tests/integration/operator/switch_cr_test.go

```diff
@@ -332,17 +332,20 @@ func verifyInstallation(t *testing.T, ctx resource.Context,
 	return K8SObjects
 }
 
-func compareInClusterAndGeneratedResources(t *testing.T, cs cluster.Cluster, k8sObjects object.K8sObjects,
+func compareInClusterAndGeneratedResources(t framework.TestContext, cs cluster.Cluster, k8sObjects object.K8sObjects,
 	expectRemoved bool) {
+	// nolint:staticcheck
```

Comment:
For some reason when I modified this to use `framework.TestContext` the linter was giving me errors for this line and the one on 348 about nil pointer use. I think it's related to a false positive in staticcheck as it would report 348 before 337.